LTS.md

```diff
@@ -0,0 +1,25 @@
+# Supported versions
+
+## Long Term Support
+
+Express Long Term Support (LTS) version is provided as per defined in this document - and also include all the packages part of the Express.js Organization (express, pillarjs and jshttp).
+
+1. Express follows a semantic versioning A.B.C (also know as semver), where the major version is represented by "A". Major version are supported for a minimal time of a year (as seen as 365 days). 
+
+2. After the time that a LTS version (that can exceed one year) is maintained, the version will move to a maintenance mode during which only security patches will be released. This maintenance mode will be supported for at least one year. 
```

Comment:
I think the Fastify doc is clearer here too, you're basically saying after a new major version is released, the old one goes into LTS and supported for 1 year?